tests/steinberger_phase_fractions_background.prm

```diff
@@ -0,0 +1,131 @@
+# This is a copy of Steinberger compressible test,
+# but it uses a pyrolite material property table instead
+# of the test data, so that reasonable seismic velocities are
+# computed. The property table also contains volume fractions of the
+# stable phases in the rock, which can be output using
+# the "named additional outputs" in the list of named outputs for
+# the Visualization postprocessor.
```

Comment:
Can you update the description to make it clear how this test is different from the one above?

Reply:
Done.